CRM/Core/BAO/CustomField.php

```diff
@@ -134,9 +134,8 @@ public static function bulkSave($bulkParams, $defaults = []) {
    * @param array $records
    * @return CRM_Core_DAO_CustomField[]
    * @throws CRM_Core_Exception
-   * @throws CiviCRM_API3_Exception
    */
-  public static function writeRecords(array $records) {
+  public static function writeRecords(array $records): array {
```

Comment:
When adding `: array` to this function, you should probably also add it to the overridden parent function.

Reply:
It might make it into an extension's BAO someday, so better to add it now.

Reply:
@colemanw done - looks like it's just one place for now so not too later on this fn - I'm less sure about `deleteRecord` which would ideally have a hint too